src/test/java/org/opensearch/security/dlic/rest/api/DashboardsInfoActionTest.java

```diff
@@ -22,12 +22,30 @@
 import org.opensearch.common.settings.Settings;
 import org.junit.Assert;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import com.google.common.collect.ImmutableList;
 
 import static org.opensearch.security.OpenSearchSecurityPlugin.LEGACY_OPENDISTRO_PREFIX;
 import static org.opensearch.security.OpenSearchSecurityPlugin.PLUGINS_PREFIX;
 
+@RunWith(Parameterized.class)
 public class DashboardsInfoActionTest extends AbstractRestApiUnitTest {
 
+    private final String ENDPOINT;
+
+    public DashboardsInfoActionTest(String endpoint){
+        ENDPOINT = endpoint;
+    }
+
+    @Parameterized.Parameters
+    public static Iterable<String> endpoints() {
+        return ImmutableList.of(
+                LEGACY_OPENDISTRO_PREFIX + "/api/securityconfig",
```

Comment:
Why is it parametrized on `securityconfig`, while testing `dashboardsinfo` and `kibanainfo`?

Reply:
I separated `securityConfig` test from the others.

Reply:
Based on test name it should test `dashboardsinfo` and `kibanainfo` API, `securityConfig` should be part of a different unit test.

Reply:
Done

Reply:
The primary focus of the `DashboardsInfoActionTest` must be `dashboardsinfo` and `kibanainfo` API, not `securityconfig`. Test must be parametrized on those two APIs, not `securityconfig` that is secondary to the test.

Reply:
but we can not parametrized on `kibanainfo` and `dashboardsinfo`. `kibanainfo` only works with `_opendistro` and `dashboardsinfo` only works with `_plugins`.

Reply:
Please explain why it is not possible to parametrize on `_plugins/dashboardsinfo` and `_opendistro/kibanainfo`?

Reply:
How should I parametrized it? for `dashboardsinfo` we only need `_plugins` and for `kibanainfo` we only need `_opendistro`, so how can I parametrized it when these two only works with one prefix?

Reply:
Can't it be parametrized on `LEGACY_OPENDISTRO_PREFIX + "/kibanainfo` and `PLUGINS_PREFIX + "/dashboardsinfo"`?

Reply:
Fixed